apps/authentication-service/src/resources/clients/clientsRoutesV1.ts

```diff
@@ -0,0 +1,88 @@
+import { clientCreateSchema } from "@/resources/clients/clientsValidation";
+import { Client, ClientCreate } from "auth0";
+import { Request, Response, Router } from "express";
+
+import { ReqValidate } from "@/middleware/validateMiddleware";
+import { validateAccessToken } from "@/middleware/authMiddleware";
+import {
+  getClient,
+  createClient,
+  deleteClient,
+} from "@/resources/clients/clientsService";
+
+const apiVersion = "v1";
+
+export const clientsRoutesV1 = (router: Router): void => {
+  router.get("/:id", [validateAccessToken], (req: Request, res: Response) => {
```

Comment:
I'd abstract these arrow functions and export them so we can test their success and failure cases